docker-compose.yml

```diff
@@ -1,32 +1,35 @@
-version: '2'
+version: '3'
+
 services:
-    image: sabaitech/seafile:latest
-    container_name: seafile
+  seafile:
+    build: .
```

Comment:
This docker-compose file is an example on how to use this image. People will download the image and not download the Dockerfile and build the image. So ot is better to keep as it was.